pkg/chain/ethereum/connect.go

```diff
@@ -122,6 +137,35 @@ func Connect(accountKey *keystore.Key, config *ethereum.Config) (*EthereumChain,
 	}, nil
 }
 
+func (ec *ethereumChain) Name() string {
+	return "Ethereum [network: " + ec.networkID + "]"
+}
+
+func (ec *ethereumChain) OperatorID() chain.OperatorID {
+	return ec.PublicKeyToOperatorID(&ec.accountKey.PrivateKey.PublicKey)
+}
+
+// combinedChainID represents all chain ids in one type: KeepMemberID,
+// OperatorID, KeepApplicationID, and KeepID. For the Ethereum chain, this is an
+// appropriate abstraction since all are underlied by the common.Address type.
+type combinedChainID common.Address
```

Comment:
Maybe this could be named `commonChainID`? IMHO `combinedChainID` would be a good fit if only `KeepMemberID` and `OperatorID` are the same. Here we're defining a common type for **all** the IDs for the chain.

Reply:
`combinedChainID` refers to the fact that it combines representation of all of the ID types under one internal type. `commonChainID` would be fine by me as well, though.